packages/docs/docs/docs/testing.md

```diff
@@ -1,18 +1,19 @@
----
+
+  ---
 id: testing
 title: Testing upgradeable applications
 sidebar_label: Testing
 ---
 
-When working with ZeppelinOS, you can test your contracts as you usually do, or you can have ZeppelinOS automatically set up your entire application in your testing environment. This allows you to replicate the same set of contracts that manage your application for each test you run.
+When working with ZeppelinOS, you can test your contracts as you usually do, or you can have ZeppelinOS automatically set up your entire project in your testing environment. This allows you to replicate the same set of contracts that manage your project for each test you run.
 
-The `zos` package provides a `TestApp()` function to retrieve your application structure from the `zos.json` file and deploy everything to the current test network. All contracts you have registered via `zos add`, plus all the contracts provided by the EVM package you have linked, will be available. The returned [`App`](https://github.com/zeppelinos/zos-lib/blob/master/src/app/App.js) object provides convenient methods for creating upgradeable instances of your contracts, which you can use for testing.
+The `zos` package provides a `TestApp()` function to retrieve your project structure from the `zos.json` file and deploy everything to the current test network. All contracts you have registered via `zos add`, plus all the contracts provided by the EVM packages you have linked, will be available. The returned [`App`](https://github.com/zeppelinos/zos-lib/blob/master/src/app/App.js) object provides convenient methods for creating upgradeable instances of your contracts, which you can use for testing.
```

Comment:
This should be `TestHelper()` instead of `TestApp()`.

Reply:
> The returned [`App`](https://github.com/zeppelinos/zos-lib/blob/master/src/app/App.js) object

`TestHelper` returns a `Project` (either a `SimpleProject` or an `AppProject`), not an `App`.